File: RokniAppApi/aspnet-core/src/RokniAppApi.Domain/Industry/Industry.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using RokniAppApi.Domain.NoteModel;
using RokniAppApi.StockModel;
using Volo.Abp.Domain.Entities.Auditing;

namespace RokniAppApi.IndustryModel
{
  public class Industry : AuditedAggregateRoot<Guid>
  {
    public string Name { get; set; }
    public virtual ICollection<Stock> Stocks { get; set; }
    public string ChartIndex { get; set; }
    public string CodalLink { get; set; }
    public string ChartsazLink { get; set; }
    public string TablokhaniLink { get; set; }
    public string TsetmcLink { get; set; }
    public string RahvardLink { get; set; }
    public string ShakhesbanLink { get; set; }
    public int SortNumber { get; set; }
    public Guid IndustryNotebookId { get; set; }
    public virtual IndustryNotebook IndustryNotebook { get; set; }
    public Industry()
    {
      Id = Guid.NewGuid();
    }
  }
}

File: AppUI/src/app/Models/Industry.ts
import { IndustryNotebookDto } from "./Notebook";
import { StockDto } from "./Stock";


export interface IndustryDto {
  id: string,
  name: string,
  codalLink: string,
  chartsazLink: string,
  tablokhaniLink: string,
  chartIndex: string,
  tsetmcLink: string,
  rahvardLink: string,
  shakhesbanLink: string,
  chartixLink:string,
  sortNumber: number,
  industryNotebookId?: string,
  choosen:boolean,
  industryNotebook: IndustryNotebookDto,
  stocks?: StockDto[]
}

export interface IndustryCreateUpdateDto {
  id?: string,
  name: string,
  codalLink: string,
  chartsazLink: string,
  chartIndex: string,
  tablokhaniLink: string,
  tsetmcLink: string,
  rahvardLink: string,
  shakhesbanLink: string,
  chartixLink:string,
  sortNumber: number,
  choosen:boolean,
  industryNotebookId: string
}